### PR TITLE
Add proper meshkit sytle error logging for ClearLine function

### LINKED ISSUE #10343: [meshercyctl] helpers.go: needs MeshKit style error
@gyohuangxin @MUzairS15 I wonder if we should be using a MeshKit Error here. https://docs.meshery.io/project/contributing/contributing-error

_Originally posted by @leecalcote in https://github.com/meshery/meshery/pull/10280#discussion_r1488901680_

### PR DESCRIPTION
**Pull request by Aijeyomah**
**Notes for Reviewers**

This PR fixes #10343

- Define error for clear command line
- Add proper meshkit sytle error logging for ClearLine function

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by RipulHandoo — DISMISSED** [review]

**Comment by RipulHandoo** [comment]
LGTM.

File: mesheryctl/pkg/utils/error.go [hunk 1]
```diff
@@ -503,3 +504,11 @@ func ErrAppendToSheet(err error, id string) error {
 		[]string{"Error occurred while appending to the spreadsheet", "The credential might be incorrect/expired"},
 		[]string{"Ensure correct append range (A1 notation) is used", "Ensure correct credential is used"})
 }
+
+func ErrClearLine(err error) error {
+	return errors.New(ErrClearLineCode, errors.Alert,
+		[]string{"Failed to clear terminal"},
+		[]string{err.Error()},
+		[]string{"Error occurred while attempting to command-line interface"},
```
Comment (leecalcote) [inline mesheryctl/pkg/utils/error.go:512]: Thanks for this PR, @Aijeyomah 👍 

Will you please rephrase this error description?

File: mesheryctl/pkg/utils/helpers.go [hunk 2]
```diff
@@ -549,7 +549,8 @@ func ClearLine() {
 	clearCmd.Stdout = os.Stdout
 	err := clearCmd.Run()
 	if err != nil {
-		log.Fatal(err)
+		Log.Error(ErrClearLine(err))
```
Comment (leecalcote) [inline mesheryctl/pkg/utils/helpers.go:552]: A MeshKit error. Excellent, @Aijeyomah

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: mesheryctl/pkg/utils/error.go [hunk 3]
```diff
@@ -38,6 +38,7 @@ var (
 	ErrRequestResponseCode    = "mesheryctl-1114"
 	ErrMarshalStructToCSVCode = "mesheryctl-1115"
 	ErrAppendToSheetCode      = "mesheryctl-1116"
+	ErrClearLineCode          = "mesheryctl-1117"
```
Comment (MUzairS15) [inline mesheryctl/pkg/utils/error.go:41]: Please do not add the error code manually, when you define a new error code leave it empty, the error code utility will auto-assign an error code.
https://docs.meshery.io/reference/error-codes

Comment (Aijeyomah) [reply mesheryctl/pkg/utils/error.go:41]: @MUzairS15 I noticed that mesheryctl error are skipped.  I looked through the CI build code for errors and saw this

`~/errorutil update -d . -i ./server/helpers -o ./server/helpers --skip-dirs mesheryctl` This line appears to be seems to take care of updating Error codes. I noticed the error code wasn't updated when i pushed an empty new error code

**Comment by RipulHandoo** [comment]
@Aijeyomah Merge conflicts.

**Comment by Aijeyomah** [comment]
> @Aijeyomah Merge conflicts.

Done

File: mesheryctl/pkg/utils/error.go (hunk 3 above)
Comment (leecalcote) [reply mesheryctl/pkg/utils/error.go]: @Aijeyomah @RipulHandoo @alphaX86 do we need to change our Makefile, update our contrib docs, or update our GitHub workflow, or something else?

**Review by RipulHandoo — APPROVED** [review]

**Review by alphaX86 — APPROVED** [review]

**Comment by alphaX86** [comment]
@Aijeyomah merge conflicts! Resolve them and we're good 👍

**Comment by gyohuangxin** [comment]
@RipulHandoo @alphaX86 @MUzairS15 @leecalcote Can we merge this PR?

**Review by leecalcote — APPROVED** [review]
Looks like quality work to me, @Aijeyomah. 👌

**Pull request merged**